db/schema.sql

```diff
@@ -390,3 +390,23 @@ CREATE TABLE `report_bitrise_builds_count` (
 ) ENGINE=InnoDB AUTO_INCREMENT=8461 DEFAULT CHARSET=utf8mb3;
 /*!40101 SET character_set_client = @saved_cs_client */;
 
+--
+-- Table structure for table `report_sentry_issue`
+--
+
+DROP TABLE IF EXISTS `report_sentry_issues`;
```

Comment:
TODO: Add `report_sentry_issues` to the `staging` and `production` tables.

Reply:
I have added `report_sentry_issues` to both `staging` and `production` databases.